### ISSUE TITLE
Unblocked subscriptions of  an API are not copied to the next version

### ISSUE THREAD

**Issue by HiranyaKavishani** [wso2/carbon-apimgt#6026]
**Description:**
Unblocked subscriptions of an API are not copied to the next version when the API has on hold subscriptions.

**Affected Product Version:**
APIM 1.10

**Steps to reproduce:**
* Create API and publish it.
* Add subscribers both of unblocking status and on hold status using custom subscription workflow 
* Add a new version of it and publish the API.
* Check the subscription list

**Comment by msm1992** [comment]
Fixed with https://github.com/wso2/carbon-apimgt/pull/6044.
